src/services/jupyternotebook.service

```diff
@@ -0,0 +1,20 @@
+# Install to ~/.config/systemd/user/jupyternotebook.service
+[Unit]
+Description=An interactive python notebook server
+After=network.target
+
+[Service]
+# This is unsafe, but GCP blocks these ports by default.
+ExecStart=/usr/bin/jupyter notebook \
+    --no-browser \
+    --port=8082 \
```

Comment:
8888 is usually the standard people are expecting.

Reply:
will change that